src/components/dialog/dialog.js

```diff
@@ -1,107 +1,293 @@
+import {keyCodes, defaultClassNames, NATIVELY_FOCUSABLE_ELEMENTS} from '../../constants';
+
```

Comment:
it was just to split the utils from the constants.  I guess we need to come up with our own conventions